packages/client/src/zkopru-node.ts

```diff
@@ -51,32 +55,23 @@ export default class ZkopruNode {
 
   async initNode(...args: any[]) {
     if (this.node) return
-    const provider = new Web3.providers.WebsocketProvider(
-      this.config.websocket as string,
-      {
-        reconnect: {
-          delay: 2000,
-          auto: true,
-        },
-        clientConfig: {
-          keepalive: true,
-          keepaliveInterval: 30000,
-        },
-      },
-    )
+
+    const provider = this.config.websocket
```

Comment:
Should we fix this like https://github.com/sifnoc/zkopru/commit/1bbd8c90c2cd978c92d42cadd8f00cdf74ed268a ?